<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleOutput</class>
 <widget class="QWidget" name="ConsoleOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>69</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>out</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="basicOutput"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ourerror</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="errorOutput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>programOut</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="programOutput"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
